handlers/Contact/components/Locations/index.js

```diff
@@ -9,7 +9,7 @@ import locations from 'locationsData';
 var renderSegment: ReactElement = (segment: Location) => (
   <TabPanel key={segment.name}>
     <div className="ContactLocations-panel">
-      <img className="ContactLocations-map" src={`https://maps.googleapis.com/maps/api/staticmap?center=${segment.addr + ' ' + segment.location}&markers=label:Skookum%7C${segment.addr}&zoom=12&size=400x400`} />
+      <div className="ContactMap"><iframe src={segment.map}></iframe></div>
```

Comment:
This change both hurts page load performance and isn’t the fully interactive map like @shieldsma wants. I propose leaving it as a static image for launch, but modifying it to link to google maps. This will give us the benefit of better UX through fast page render time and defer the extra work a fully dynamic map will entail until a better time.